src/guides/v2.3/test/static/static-analysis.md

```diff
@@ -0,0 +1,80 @@
+---
+group: testing
+title: Setting up static analysis
+functional_areas:
+  - Testing
+  - test
+---
+
+## Introduction
+
+This document provides setup steps to get most static analyzers that are used in our build pipeline working locally in PHPStorm.  This guide is written specifically for those using PHPStorm on MacOS.
+
+## Before you begin
+
+If you haven't already, make sure that node and npm are installed. Then, in the Magento root directory, run `npm install`.  After installation, ensure there is a `node_modules` directory in the root of your project before proceeding.
```

Comment:
```suggestion
If you haven't already, make sure that Node and NPM are installed. Then, in the Magento root directory, run `npm install`.  After installation, ensure there is a `node_modules` directory in the root of your project before proceeding.
```

Reply:
Those are generally accepted as lowercase.

Reply:
In that case (please add @hguthrie's suggestion for emphasis)